README.md

```diff
@@ -1,2 +1,79 @@
-# maestro
-Indexing software to connect SONG and Arranger components of the overture stack 
+# Maestro
+
+The indexer component of genomic studies metadata
+
+## Intro
+Meastro was created to enable genomic researchers to enhance their Overture SONGs by building indexes, elastic search
+by default, that makes searching Analyses and Studies much more powerful and easier.
+
+
+### Features:
+- Supports indexing from multiple metadata repositories (SONG).
+- Multiple indexing requests: analysis, study, full repository.
+- Event driven indexing.
+- Ability to Exclude analysis based on different Ids: Study, Analysis, Donor, Sample Or file.
+
+### Design Goals:
+- Reactive
+    - Event driven
+    - Elastic
+    - Resiliency & Fault tolerence
+- Failure audit
+- Runtime configurability
+- Extendable: separate domain from infrastructure & configuration
+
+## Tecnologies & libraries:
+- Java 11 - OpenJDK 11
+- Maven 3
+- lombok
+- Spring boot 2
+    - Spring webflux & project reactor
+    - Spring data elasticsearch
+    - Spring retry
+    - Spring Cloud
+        - Streams (for Kafak integration)
+        - config client
+    - Spring boot admin + client
+- Elasticsearch 6.6.1
+- Apache Kafka
+- resilience4j:
+    - retry module
+- vavr 
+- Testing libraries:
+    - Junit 5
+    - Mockito 2
+    - testcontainers
+    - Spring cloud contract wiremock
+
+## Structure
+The project is following the ports/adapters archeticture, where the domain is completely isolated from external infrastructure
+and frameworks.
+- Two maven modules:
+    - maestro domain
+      - the core features logic, framework independent portable that contains the main indexing, rules, notifications
```

Comment:
reword this `- the core features logic, framework independent portable that contains` to something like `- core features and framework independent logic that is portable and contains ...`